perun/utils/view_helpers.py

```diff
@@ -0,0 +1,206 @@
+"""Collection of helper functions for working with bokeh graphs"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Any, Union
+from types import ModuleType
+from collections.abc import MutableMapping, Iterable
+
+from enum import Enum
+import demandimport
+
+with demandimport.enabled():
+    from bokeh.plotting import show, output_file
+    import bokeh.palettes as bk_palettes
+    import bokeh.themes.theme as bk_theme
+    import holoviews as hv
+
+import perun.profile.helpers as profiles
+from perun.utils import log
+
+if TYPE_CHECKING:
+    import pandas as pd
+    from perun.profile.factory import Profile
+
+
+__author__ = "Tomas Fiedor"
```

Comment:
Ehm.

Reply:
Although Git shows this code as my addition, the file was originally called bokeh_helpers.py and this line was authored by you :) . I just updated and renamed the file to better suit the Holoviews needs. On a side note, I feel like the __author__ tags on a per-file basis are becoming a bit obsolete and unnecessary maybe? Maintaining those "tags" requires manual effort and is generally error-prone (a lot of modules were updated or changed by multiple people over time). Why not simply rely on git blame to deduce authorship on a line by line basis?

Reply:
Yes, I am thinking the same, I think the issue is that it was automatically added at the begining, I kept it, and rest of you simply copied this :D.